petric.py

```diff
@@ -312,7 +312,9 @@ def get_image(fname):
         (SRCDIR / "Mediso_NEMA_IQ", OUTDIR / "Mediso_NEMA",
          [MetricsWithTimeout(outdir=OUTDIR / "Mediso_NEMA", **DATA_SLICES['Mediso_NEMA_IQ'])]),
         (SRCDIR / "GE_DMI3_Torso", OUTDIR / "DMI3_Torso",
-         [MetricsWithTimeout(outdir=OUTDIR / "DMI3_Torso", **DATA_SLICES['GE_DMI3_Torso'])])]
+         [MetricsWithTimeout(outdir=OUTDIR / "DMI3_Torso", **DATA_SLICES['GE_DMI3_Torso'])]),
+        (SRCDIR / "Siemens_Vision600_Hoffman", OUTDIR / "Vision600_Hoffman",
```

Comment:
```suggestion
        (SRCDIR / "Siemens_Vision600_Hoffman", OUTDIR / "Siemens_Vision600_Hoffman",
```
I think in general it'd be far more sensible to use the same name all the time.